File: 2938-separate-black-and-white-balls/2938-separate-black-and-white-balls.py
class Solution:
    def minimumSteps(self, s: str) -> int:
        white = 0
        ans = 0
        n = len(s)
        for j in range(n):
            if s[j] == '0':
                ans += white
            if s[j] == '1':
                white += 1
        return ans

File: LeetCode/2938-separate-black-and-white-balls/2938-separate-black-and-white-balls.cpp
class Solution {
public:
    long long minimumSteps(string s) {
        int white = 0;
        long long ans = 0;
        int n = s.size();
        
        for (int j = 0; j < n; j++) {
            if (s[j] == '0') {
                ans += white;
            }
            if (s[j] == '1') {
                white++;
            }
        }
        
        return ans;
    }
};